src/openklant/conf/base.py

```diff
@@ -415,21 +415,15 @@
 AXES_CACHE = "axes"  # refers to CACHES setting
 AXES_FAILURE_LIMIT = 5  # Default: 3
 AXES_LOCK_OUT_AT_FAILURE = True  # Default: True
-AXES_USE_USER_AGENT = False  # Default: False
 AXES_COOLOFF_TIME = datetime.timedelta(minutes=5)
 # after testing, the REMOTE_ADDR does not appear to be included with nginx (so single
 # reverse proxy) and the ipware detection didn't properly work. On K8s you typically have
 # ingress (load balancer) and then an additional nginx container for private file serving,
 # bringing the total of reverse proxies to 2 - meaning HTTP_X_FORWARDED_FOR basically
 # looks like ``$realIp,$ingressIp``. -> to get to $realIp, there is only 1 extra reverse
 # proxy included.
-AXES_PROXY_COUNT = NUM_PROXIES - 1 if NUM_PROXIES else None
-AXES_ONLY_USER_FAILURES = (
-    False  # Default: False (you might want to block on username rather than IP)
-)
-AXES_LOCK_OUT_BY_COMBINATION_USER_AND_IP = (
-    False  # Default: False (you might want to block on username and IP)
-)
+AXES_IPWARE_PROXY_COUNT = NUM_PROXIES - 1 if NUM_PROXIES else None
+AXES_LOCKOUT_PARAMETERS = ["ip_address"]
```

Comment:
@alextreme I noticed that these settings are not the same across different components:

- Open Zaak and Open Klant use `["ip_address"]` (or the equivalent for version 5.x.x)
- Open Forms and Objecttypes use `[["ip_address", "user_agent", "username"]]`

Reply:
@stevenbal default-project uses `AXES_LOCKOUT_PARAMETERS = ["username", "email", "ip"]`, but this may have to do with newer versions of django-axes. As long as the number of attempts is sufficient and the cooldown time isn't too long it shouldn't matter too much, doing blocking at least only on ip_address would have my preference than on the combination with UA + username